homeassistant/components/generic/strings.json

```diff
@@ -55,6 +55,7 @@
           "authentication": "[%key:component::generic::config::step::user::data::authentication%]",
           "limit_refetch_to_url_change": "[%key:component::generic::config::step::user::data::limit_refetch_to_url_change%]",
           "password": "[%key:common::config_flow::data::password%]",
+          "use_wallclock_as_timestamps": "Rewrite timestamps",
```

Comment:
Is it possible to provide slightly more explanation of what this is?  How does the user decide what to set it to and can we hint at this in the interface somehow?

Reply:
I don't know, do you have any ideas on what could be clarified in a few words? I wrote a little more about the problem in https://github.com/home-assistant/core/pull/64295
Most users don't need it. The ones that need it will figure it out. Maybe just say that default is off, and maybe some notes in the docs that say what can be used for.

Reply:
There are at least two different cases that it may help with: 1) fixing segmenting issues in videos with audio and 2) fixing sporadic crashes from some camera types, but I'm not sure how to combine those both into the option description. Why not just put what it does in the documentation? Or just change the name back to "Use wallclock as timestamps" and let them google it and figure it out for themselves.
I understand there is a desire to make HA easier to use, but I think trying to idiot proof everything is overkill. If they're going to flip every option then let them do it and see what happens. Playing with that option shouldn't really affect things for most users anyway no matter what the setting.

Reply:
Sure, sounds good